src/RemoteStorageDetails/RemoteStorageDetails.js

```diff
@@ -144,6 +144,13 @@ const RemoteStorageDetails = ({
                 </KeyValue>
               </Col>
             )}
+            {storage.apiKey && (
```

Comment:
interesting. In Aquisitions we have pattern `<KeyValue>{{value || <NoValue />}}</KeyValue>`, but it might be different in scope of your requirements of course

Reply:
In this case, we don't need to show this part at all

Reply:
mb Steph doesn't know about common pattern :)

Reply:
The is 3 types of provider and each of them has different fields, I don't see any reason to show NoValue if there can't be any values in anyway